src/components/Card.js

```diff
@@ -27,10 +49,19 @@ export class Card {
   _setEventListeners() {
     this._templateClone
       .querySelector(".card__button")
-      .addEventListener("click", this._likeCard);
+      .addEventListener("click", (event) => {
+        this.handleLike(this.id, this.isLiked)
+          .then((res) => res.json())
+          .then((data) => console.log(data));
```

Comment:
You need to `catch` possible errors at the very **end** of any server request.

**Please, correct it everywhere in the code**